kafka-scheduler/src/main/java/com/mesosphere/dcos/kafka/config/KafkaSchedulerConfiguration.java

```diff
@@ -113,21 +128,20 @@ public boolean equals(Object o) {
         if (this == o) {
             return true;
         }
-
         if (o == null || getClass() != o.getClass()) {
             return false;
         }
-
         KafkaSchedulerConfiguration that = (KafkaSchedulerConfiguration) o;
-        return Objects.equals(serviceConfiguration, that.serviceConfiguration) &&
-                Objects.equals(brokerConfiguration, that.brokerConfiguration) &&
-                Objects.equals(executorConfiguration, that.executorConfiguration) &&
-                Objects.equals(kafkaConfiguration, that.kafkaConfiguration);
+        return com.google.common.base.Objects.equal(serviceConfiguration, that.serviceConfiguration) &&
+                com.google.common.base.Objects.equal(brokerConfiguration, that.brokerConfiguration) &&
+                com.google.common.base.Objects.equal(kafkaConfiguration, that.kafkaConfiguration) &&
+                com.google.common.base.Objects.equal(executorConfiguration, that.executorConfiguration) &&
+                com.google.common.base.Objects.equal(repairConfiguration, that.repairConfiguration);
```

Comment:
We're never going to use anything prior to Java 7.  Migration to Guava Objects is unnecessary.